app/src/main/java/com/yacgroup/yacguide/TourbookActivity.kt

```diff
@@ -101,18 +94,11 @@ class TourbookActivity : BaseNavigationActivity() {
     public override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         if (resultCode == IntentConstants.RESULT_UPDATED) {
             Toast.makeText(this, R.string.ascend_deleted, Toast.LENGTH_SHORT).show()
-        }
-    }
-
-    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<String>, grantResults: IntArray) {
-        if (FilesystemUtils.permissionGranted(grantResults)) {
-            _showFileChooser()
-        } else {
-            val errorTextId = if (_ioOption == IOOption.eImport)
-                    R.string.import_impossible
-                else
-                    R.string.export_impossible
-            Toast.makeText(this, errorTextId, Toast.LENGTH_SHORT).show()
+        } else if (resultCode == Activity.RESULT_OK) {
```

Comment:
These resultCodes are just integers. Can you ensure that `Activity.RESULT_OK` differs from the codes defined in `IntentConstants`?

Reply:
I cannot directly guarantee that but according to the documentation `RESULT_OK := -1`.